server/service/key/service_key.go

```diff
@@ -51,7 +51,11 @@ func (s *secretKeyService) GetSecretKey(ctx context.Context, req *key.GetSecretK
 
 	sharedKey := crypto.DeriveSharedKey(oraclePrivKey, consumerPubKey, crypto.KDFSHA256)
 
-	secretKey := GetSecretKey(oraclePrivKey.Serialize(), dealID, req.DataHash)
+	dataHashBz, err := hex.DecodeString(req.DataHash)
+	if err != nil {
+		fmt.Errorf("failed to decode dataHash(%s). %w", req.DataHash, err)
```

Comment:
I think you're missing a return statement :)

Reply:
OK. I fixed it.(https://github.com/medibloc/panacea-oracle/pull/94/commits/56e5ca3d0b078fbded9416adf60f6098714387ba) 
Thanks